commands/dns_cli.py

```diff
@@ -321,15 +321,22 @@ def check_r53(domain_session, project_session, domain, base_domain):
     return cert_arn
 
 
-def lb_domain(project_session: Session, app: str, svc: str, env: str) -> Tuple[str, dict]:
+# Todo: Maybe rename method to something which describes its function?
+def get_elastic_load_balancer_domain_and_configuration(
```

Comment:
Renamed to try and better explain the method's expected behaviour.

Reply:
technically it's not an elastic load balancer, it's an application load balancer.

Reply:
Done